test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -201,55 +201,6 @@ public void EntryPointCaching()
             Done();
         }
 
-        //[Fact]
```

Comment:
Thanks @TomFinley I'll take a look. Should my rule of thumb be that if it compiles, I leave it as fact-skip, otherwise I remove it? I could try to see what things are renamed to, but I'm still not super familiar with this project yet

Reply:
It *may* be that it's appropriate to remove it, but I'd prefer to do so in a more deliberative way.

---
In reply to: [201912980](https://github.com/dotnet/machinelearning/pull/514#discussion_r201912980) [](ancestors = 201912980)

Reply:
I have re-added those tests, and just left them commented out. I tried to simply change Dv1 to DvInt1, but the parameters are different. I'd rather somebody with a greater understanding of what is going on here dig through it. Thanks for the feedback!